test/integration/auth/mongodb_aws.test.ts

```diff
@@ -81,6 +87,35 @@ describe('MONGODB-AWS', function () {
     expect(provider).to.be.instanceOf(MongoDBAWS);
   });
 
+  describe('with missing aws token', () => {
+    let awsSessionToken;
+
+    beforeEach(function () {
+      awsSessionToken = process.env.AWS_SESSION_TOKEN;
+      delete process.env.AWS_SESSION_TOKEN;
+    });
+
+    afterEach(async () => {
+      process.env.AWS_SESSION_TOKEN = awsSessionToken;
```

Comment:
```suggestion
      if (awsSessionToken != null) process.env.AWS_SESSION_TOKEN = awsSessionToken;
```

if AWS_SESSION_TOKEN did not exist then this would set it to `"undefined"`